File: TI2.1  SOAP_JAVA_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_soap_java/src/main/java/ec/edu/monster/servicio/LoginService.java
package ec.edu.monster.servicio;

public class LoginService {

    public boolean login(String username, String password) {
        if ((username.equals("MONSTER") && password.equals("MONSTER9"))
                || (username.equals("admin") && password.equals("admin"))) {
            return true;
        }
        return false;
    }
}

File: Tl2.2 SOAP_DOTNET_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_soap_dotnet/ec.edu.monster.service/LoginService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_eureka_bank_soap_dotnet.ec.edu.monster.service
{
    public class LoginService
    {
        public bool Login(string username, string password)
        {
            return (username == "MONSTER" && password == "MONSTER9") ||
                   (username == "admin" && password == "admin");
        }
    }
}
